demisto_sdk/commands/find_dependencies/tests/find_dependencies_test.py

```diff
@@ -1,24 +1,294 @@
+
+    # Create a pack called 'Logzio' with 1 incident_type.
+    logzio = repo.create_pack('Logzio')
+    create_a_pack_entity(logzio, FileType.INCIDENT_TYPE, 'Logz.io Alert', 'Logz.io Alert')
+
+    # Create a pack called 'AccessInvestigation' with 1 incident_type.
+    access_investigation = repo.create_pack('AccessInvestigation')
+    create_a_pack_entity(access_investigation, FileType.INCIDENT_TYPE, 'Access', 'Access')
+
+    # Create a pack called 'PrismaCloud' with 1 incident_type.
+    prisma_cloud = repo.create_pack('PrismaCloud')
+    create_a_pack_entity(prisma_cloud, FileType.INCIDENT_TYPE, 'AWS CloudTrail Misconfiguration',
+                         'AWS CloudTrail Misconfiguration')
+
+    # Create a pack called 'BruteForce' with 1 incident_field.
+    brute_force = repo.create_pack('BruteForce')
+    create_a_pack_entity(brute_force, FileType.INCIDENT_FIELD, 'accountgroups', 'Account Groups')
+
+    # Create a pack called 'Compliance' with 1 incident_field.
+    complience = repo.create_pack('Compliance')
+    create_a_pack_entity(complience, FileType.INCIDENT_FIELD, 'emailaddress', 'E-mail Address')
+
+    # Create a pack called 'CortexXDR' with 1 classifier.
+    cortex_xdr = repo.create_pack('CortexXDR')
+    create_a_pack_entity(cortex_xdr, FileType.CLASSIFIER, 'Cortex XDR - IR', 'Cortex XDR - IR')
+
+    # Create a pack called 'ImpossibleTraveler' with:
+    # 1 integration 1 playbook 1 test_playbook 1 layout 7 incident_fields 1 incident type
+    impossible_traveler = repo.create_pack('ImpossibleTraveler')
+    create_a_pack_entity(impossible_traveler, FileType.SCRIPT, 'CalculateGeoDistance')
+    create_a_pack_entity(impossible_traveler, FileType.PLAYBOOK, 'Impossible Traveler')
+    create_a_pack_entity(impossible_traveler, FileType.TEST_PLAYBOOK, 'Impossible Traveler - Test')
+    create_a_pack_entity(impossible_traveler, FileType.LAYOUT, 'Impossible Traveler')
+    create_a_pack_entity(impossible_traveler, FileType.INCIDENT_FIELD, 'coordinates' 'Coordinates')
+    create_a_pack_entity(impossible_traveler, FileType.INCIDENT_FIELD, 'previouscoordinates' 'Previous Coordinates')
+    create_a_pack_entity(impossible_traveler, FileType.INCIDENT_FIELD, 'previouscountry' 'Previou Country')
+    create_a_pack_entity(impossible_traveler, FileType.INCIDENT_FIELD,
+                         'previoussignindatetime' 'Previous Sign In Date Time')
+    create_a_pack_entity(impossible_traveler, FileType.INCIDENT_FIELD, 'previoussourceiP' 'Previous Source IP')
+    create_a_pack_entity(impossible_traveler, FileType.INCIDENT_FIELD, 'signindatetime' 'Sign In Date Time')
+    create_a_pack_entity(impossible_traveler, FileType.INCIDENT_FIELD, 'travelmaplink' 'Travel Map Link')
+    create_a_pack_entity(impossible_traveler, FileType.INCIDENT_TYPE, 'impossibletraveler' 'Impossible Traveler')
+
+    with ChangeCWD(repo.path):
+        ids = cis.IDSetCreator()
+        ids.create_id_set()
+
+
+def setup_module():
```

Comment:
This function creates the content repo just for the current file and removes the repo after all UT are completed, It's something like this
    
    def setup_class(cls):
        create_content_repo()

    def teardown_class(cls):
        remove_content_repo()

but instead of creating and removing the content repo for each class it only removes after it finishes all the classes of the UT file (the UT file contains some TestClasses)

[link to pytest documentation](https://docs.pytest.org/en/6.2.x/xunit_setup.html)

Reply:
Don't you need `teardown_module` as well?

Reply:
It seems not. The files are **automatically** deleted after all tests are completed